teams.txt

```diff
@@ -4,3 +4,7 @@
 [ GitHub repository link ]
 [ Heroku/Deployment link ]
 
+J Garrett Collinson, Matt Bruder, David Perez
+YouGoal
+YouGoal is a soccer player and team rating app toolkit used to look through and rate different famous and important soccer players and teams from around the world. Users ratings of individual players and teams will be compared to the current FIFA ratings of the players and teams.
+https://github.com/sharstream/YoUGoal
```

Comment:
Add your deployment link.  You can just create the app on Heroku (or wherever you want to deploy) and give us that link.